app/uploaders/claim_documentation/pension_burial/workflow.rb

```diff
@@ -6,11 +6,13 @@
 require 'workflow/task/pension_burial/upload'
 
 # Convert uploaded files for Pension/Burial to a format, stamping them twice
-# before transfering them over to the SFTP server.
+# before transferring them over to the SFTP server.
 
 class ClaimDocumentation::PensionBurial::Workflow < Workflow::File
   run Workflow::Task::Shared::ConvertToPdf
-  run Workflow::Task::Shared::DatestampPdfTask, text: 'Vets.gov', x: 0, y: 0
+  # The timestamp in this task is UTC, but the business requirement is for CST.
+  # Datetime will be added to the append text.
+  run Workflow::Task::Shared::DatestampPdfTask, text: 'Vets.gov', x: 5, y: 5, text_only: true
```

Comment:
Ugh, no, you're right. The logic only does the append if there's a timestamp. Going to change that now.